### PR TITLE
Fix hooks docs

### PR DESCRIPTION
**Pull request by raphinesse**
This PR fixes various outdated parts of the hooks docs and adds a few minor improvements.

Related: https://github.com/apache/cordova-lib/pull/709, https://github.com/apache/cordova-lib/pull/710

### CODE REVIEW

File: www/docs/en/9.x/guide/appdev/hooks/index.md [hunk 1]
```diff
@@ -383,6 +377,15 @@ If a script returns a non-zero exit code, then the parent cordova command will b
...
 
+#### Windows Quirks
+
+If you are working on Windows, and your hook scripts aren't `*.bat` files, Cordova CLI will expect a shebang line as the first line of the script. This way it knows the interpreter it needs to use to launch the script. A shebang line for a Python script could look like this:
+
+```
+#!/usr/bin/env python
+```
+
+
```
Comment (erisu) [inline www/docs/en/9.x/guide/appdev/hooks/index.md]: Don't need two newlines, but also not necessary to make any effort to change this.

**Comment by raphinesse** [comment]
@erisu Do we have to do something to make these changes go live?

**Comment by erisu** [comment]
@raphinesse I can start releasing the changes now.

**Comment by raphinesse** [comment]
@erisu Great, thanks!

**Pull request merged**